### ISSUE TITLE
Configuration on PKUMMD2 dataset

### ISSUE THREAD

**Issue by JHang2020** [HuiGuanLab/HiCo#2]
Hello~ Thanks for your excellent work!
I want to confirm that the model is trained only 450 epochs on PKUMMD2 dataset. Or some more epochs? Could you share me the training settings on PKUMMD2 dataset for the paper results? 
Highly appreciate if you could help.

**Comment by ssk1997** [comment]
The configuration is the same for NTU60 and PKUMMD, including learning rate, training epochs, queue size and so on. @JHang2020
